build.gradle

```diff
@@ -120,7 +120,7 @@ subprojects { subproject ->
 		smack3Version = '3.2.1'
 		smackVersion = '4.0.6'
 		springAmqpVersion = project.hasProperty('springAmqpVersion') ? project.springAmqpVersion : '1.5.0.BUILD-SNAPSHOT'
-		springDataMongoVersion = '1.7.0.M1'
+		springDataMongoVersion = '1.7.0.RC1'
```

Comment:
Great! 
Thank you!
Now I can pull it locally and play.
FYI the Fowler RELEASE is here already: https://spring.io/blog/2015/03/23/spring-data-release-train-fowler-goes-ga
Hence we can upgrade just to `1.7.0.RELEASE`